library/system/src/modules/Initrd.rb

```diff
@@ -400,9 +400,9 @@ def Write
         path(".target.bash"),
         Builtins.sformat(
           "/sbin/mkinitrd %1 %2 >> %3 2>&1",
-          param,
-          @additional_parameters,
-          Ops.add(Directory.logdir, "/y2logmkinitrd")
+          param, # cannot escaped due multiple possible params
```

Comment:
English: "due **to**"  (also below)

Reply:
Apart from that, I don't understant that comment at all. Examples might help.

Reply:
well, I want to say that param can contant multiple parameters like "-s something" at https://github.com/yast/yast-yast2/pull/872/files/7f355a06aa1746e0c11fb8315ca9bf2bd655f296#diff-585c9223748698b81e5bb470365c90b8R397